yts/test_suite_test.go

```diff
@@ -89,30 +89,30 @@ func getEvents(in []byte) (string, error) {
 	return yaml.ParserGetEvents(in)
 }
 
+func mustRead(t *testing.T, path, name string) []byte {
+	data, err := os.ReadFile(filepath.Join(path, name))
```

Comment:
Please add `t.Helper()`
```suggestion
	t.Helper()
	
	data, err := os.ReadFile(filepath.Join(path, name))
```